doc/source/examples/sharp-immersed-boundary-solver/3d-composite-mixer-with-pbt-impeller/3d-composite-mixer-with-pbt-impeller.rst

```diff
@@ -78,16 +78,27 @@ The section defining each parameter for the particles has certains requirements:
 .. code-block:: text
 
     subsection particles
-      set number of particles                           = 1
-      set stencil order                                 = 2
-      set refine mesh inside radius factor              = 0.0
-      set refine mesh outside radius factor             = 1.1
-      set length ratio                                  = 3
-      set initial refinement                            = 3
-      set integrate motion                              = false
-      set assemble Navier-Stokes inside particles       = false
-
+      set number of particles                     = 1
+      set assemble Navier-Stokes inside particles = false
+      subsection extrapolation function
+        set stencil order = 2
+        set length ratio  = 3
+      end
+      subsection local mesh refinement
+        set initial refinement                = 3
+        set refine mesh inside radius factor  = 0.0
+        set refine mesh outside radius factor = 1.1
+      end
+      subsection output
+        set enable extra sharp interface vtu output field = true
+      end
       subsection particle info 0
+        subsection position
+          set Function expression = -0.25;0;0
```

Comment:
Is it normal that the position changed?

Reply:
That was a very nice catch. This is an error. Thanks for pointing this out.